backend/app/integrations/router.py

```
@@ -0,0 +1,275 @@
+    StartImportResponse,
+)
+from app.integrations.service import ImportService
+from fastapi import APIRouter, Depends, HTTPException, status
+from motor.motor_asyncio import AsyncIOMotorDatabase
+from splitwise import Splitwise
+
+router = APIRouter(prefix="/import", tags=["import"])
+
+
+@router.get("/splitwise/authorize")
+async def get_splitwise_oauth_url(current_user=Depends(get_current_user)):
```

Ruff (0.14.10)

Unused function argument: `current_user`

(ARG001)